boards/common/samd21-arduino-bootloader/Makefile.dep

```diff
@@ -1,5 +1,7 @@
-# This boards requires support for Arduino bootloader.
-USEMODULE += usb_board_reset
-USEMODULE += stdio_cdc_acm
+include $(RIOTBOARD)/common/makefiles/stdio_cdc_acm.dep.mk
```

Comment:
same question here.

Reply:
it's independent. You can have CDC ACM without the bootloader reset, but you can't have the bootloader reset without CDC ACM.